uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -36,98 +36,129 @@ private static function _checkRelatedExtensions()
             self::$_relatedExtensions['appraisals'] = true;
         }
     }
-    
+
+    /**
+     * Gets the contacts starting with the given ids
+     *
+     * It also chains a call to the 'Email' api and puts the value in
+     * the 'email' property
+     *
+     * @param {Array} $contactIds
+     * @return {Array}
+     */
+    private static function _getContactsWithEmail($contactIds) {
+        $contacts = civicrm_api3('Contact', 'get', array(
+            'return' => 'display_name',
+            'id' => array('IN' => $contactIds),
+            'api.Email.getsingle' => array(
+                'contact_id' => "\$value.contact_id",
+                'is_primary' => true,
+                'return' => array('contact_id', 'email')
+            )
+        ))['values'];
```

Comment:
There could be a php version compatibility issue here. Please can we change it like this:

```
$result = civicrm_api3('Contact', 'get', array(
    'return' => 'display_name',
    'id' => array('IN' => $contactIds),
    'api.Email.getsingle' => array(
    'contact_id' => "\$value.contact_id",
    'is_primary' => true,
    'return' => array('contact_id', 'email')
  )
));

$contacts = $result['values'];
```